src/components/display/Table.test.tsx

```diff
@@ -92,7 +94,7 @@ describe('Table', () => {
 		);
 		const row1 = getRowByColumnLabel('row1col1');
 		const row1Checkbox = within(row1).getByTestId(SELECTORS.checkbox);
-		expect(row1Checkbox).toBeInTheDocument();
+		expect(row1Checkbox).toBeVisible();
 		expect(row1Checkbox).toHaveStyleRule('display', 'none');
```

Comment:
in this test we are checking actually that the element is in the document but not visible (display css property set to none). I think we could update the test to be
```suggestion
		expect(row1Checkbox).toBeInTheDocument();
		expect(row1Checkbox).not.toBeVisible();
```